db/re/instance_db.txt

```diff
@@ -23,3 +23,11 @@
 16,Horror Toy Factory,3600,300,1@xm_d,111,22
 17,Faceworm's Nest,3600,300,1@face,112,374
 18,Ghost Palace,3600,300,1@spa,42,196
+19,Bios Island,7200,300,1@dth1,19,94,1@dth2,1@dth3
```

Comment:
From my data Bios Island (or Isle of Bios) instance duration is 3600s, entrance at 17,93